src/test/cdk/cdkExplorer.test.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import { AwsCdkExplorer } from '../../cdk/explorer/awsCdkExplorer'
+
+describe('AwsCdkExplorer', () => {
+    it('displays no nodes in empty workspace', async () => {
```

Comment:
This test assumes there is no open workspace. I'd wrap this `it` block within a check, so that for now the test is skipped if VS Code happens to have a workspace open.

Even better, you could use sinon to stub the vs code api to get workspaces to return undefined for this test. Then you are unconditionally testing this scenario.

Reply:
done